app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_results = {}
+
+    def wrapper(*args) -> Any:
+        if args in cache_results:
+            print("Getting from cache")
+            return cache_results[args]
```

Comment:
Move it after if-else
See example:
https://github.com/mate-academy/py-cache-decorator/blob/master/checklist.md#dont-repeat-yourself